cloudferrylib/os/actions/verify_vms.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils as utl
+
+
+class VerifyVms(action.Action):
+    def __init__(self, init, cloud=None):
```

Comment:
I guess we can get rid of this `__init__` method, cause there are no any actions here (and just use `__init__` of parent class)